exanauts/Hiop.jl
# Minor updates for Hiop

This PR implements a few things, which, I hope, will improve the package overall: 
- convert all indents to 4 spaces, to homogenize formatting across the package (I tried to follow [JuMP's conventions](www.juliaopt.org/JuMP.jl/dev/style/))
- allow to run test via `test Hiop` 
- update build procedure to load Hiop on MacOS (should resolve #1 )
- update the README accordingly
